pygame/README.md

```diff
@@ -1,25 +1,62 @@
-This folder contains some simple camera classification and detection examples using pygame
+# P examples for inferencing on Coral
```

Comment:
"P" examples ?

Reply:
This isn't the latest change. GitHub CL review tools are bad, I know.